bundle.yaml

```diff
@@ -1,33 +1,33 @@
+name: kubeflow
 bundle: kubernetes
 applications:
-  admission-webhook:         { charm: cs:admission-webhook-10,        scale: 1 }
-  argo-controller:           { charm: cs:argo-controller-51,          scale: 1 }
-  dex-auth:                  { charm: cs:dex-auth-60,                 scale: 1 }
-  istio-ingressgateway:      { charm: cs:istio-ingressgateway-20,     scale: 1 }
-  istio-pilot:               { charm: cs:istio-pilot-20,              scale: 1, options: { default-gateway: "kubeflow-gateway" } }
-  jupyter-controller:        { charm: cs:jupyter-controller-56,       scale: 1 }
-  jupyter-ui:                { charm: cs:jupyter-ui-10,               scale: 1 }
-  katib-controller:          { charm: cs:katib-controller-30,         scale: 1 }
+  admission-webhook:         { charm: admission-webhook,       channel: edge, scale: 1 }
+  argo-controller:           { charm: argo-controller,         channel: edge, scale: 1 }
+  dex-auth:                  { charm: dex-auth,                channel: edge, scale: 1 }
+  istio-ingressgateway:      { charm: istio-ingressgateway,    channel: edge, scale: 1 }
```

Comment:
This should be pointing to `istio-gateway` with `kind: ingress`

Reply:
You can check the latest changes in notebook-operator to see an example of how the bundle should look

Reply:
Done